sdk/identity/identity/test/internal/identityClient.spec.ts

```diff
@@ -2,11 +2,11 @@
 // Licensed under the MIT license.
 
 import assert from "assert";
-import { assertRejects } from "./authTestUtils";
-import { MockAuthHttpClient } from "./authTestUtils";
-import { AuthenticationError } from "../src/";
-import { IdentityClient } from "../src/client/identityClient";
-import { ClientSecretCredential } from "../src";
+import { assertRejects } from "../authTestUtils";
+import { MockAuthHttpClient } from "../authTestUtils";
+import { AuthenticationError } from "../../src";
+import { IdentityClient } from "../../src/client/identityClient";
+import { ClientSecretCredential } from "../../src";
```

Comment:
Can we combine the multiple imports coming from the same source into a single import?

Reply:
Sure!